js/btcmarkets.js

```diff
@@ -444,9 +544,11 @@ module.exports = class btcmarkets extends Exchange {
             throw new ArgumentsRequired (this.id + ': fetchOrders requires a `symbol` argument.');
         }
         await this.loadMarkets ();
-        let market = this.market (symbol);
-        let request = this.createPaginatedRequest (market, since, limit);
-        let response = await this.privatePostOrderHistory (this.extend (request, params));
+        const market = this.market (symbol);
+        const request = this.createPaginatedRequest (market, since, limit);
+        const response = await this.privatePostOrderHistory (this.extend (request, params));
+        if (!response['success'])
```

Comment:
This should go to `handleErrors`, see here: https://github.com/ccxt/ccxt/blob/master/js/therock.js#L1177